server/server.go

```diff
@@ -61,32 +61,45 @@ func NewDefaultKeystore() *DefaultKeystore {
 // into a crypto.Signer, which is stored in the Keystore.
 func NewKeystoreFromDir(dir string, LoadKey func([]byte) (crypto.Signer, error)) (Keystore, error) {
```

Comment:
Do we still need this?
I can have client/remote_test.go and tests/common_test.go also use AddFromDir, then there would be no reason to keep this around.

Reply:
This project contains both a library and an application which uses the library. By leaving this in place we add the new functionality we need without needlessly breaking the library for anyone who was happily using the existing function.